### DISCUSSION TITLE
GitHub crashing on latest update

### DISCUSSION THREAD

**Discussion by mallowigi** [community/community#109554] [Mobile] [answered]
### Select Topic Area

Bug

### Body

Since today, I can no longer open the application (ios 16.2). I tried deleting and reinstalling the app but to no success.

This is very problematic as GitHub is part of my workflow. Fortunately I can navigate GitHub.com but it's still a major annoyance.

**Comment by alcere** [comment]
@mallowigi We are very sorry for the trouble! Could you please tell us which mobile platform you are using: Android or iOS?

**Reply by mallowigi** [reply]
I just wrote it on the ticket: ios 16.2

**Reply by taki-on** [reply]
@mallowigi sorry for the inconvenience. We shipped a fix to our Beta testers. Would you mind joining our Beta test program and seeing if it fixes your issues? You can join the program via https://testflight.apple.com/join/NLskzwi5. The latest beta version 1.151.0 (124277049) has the fix. 🙇

**Reply by mallowigi** [reply]
@taki-on not working either

**Reply by mallowigi** [reply]
https://github.com/community/community/assets/5015756/34ee49aa-c378-43a1-8e76-9a0e3f2ce4e2

**Reply by taki-on** [reply]
I am sorry for the problem again and appreciate your quick feedback! We cannot reproduce the issue on our side, so it'd be great if you can help us 🙇 In the TestFlight app, you can install _previous_ versions. Can you try out a previous version like `1.149.0` and see if it crashes? We think it has been happening since `1.150.0` but we want to confirm.

We really appreciate your help to debug these crashes.

**Reply by mallowigi** [reply]
Confirmed that it worked before 1.150.0

**Reply by taki-on** [reply]
Thank you for helping us @mallowigi . We pushed one more beta build, 1.151.0 (124542235). The build fixes crashes we can reproduce but we'd like to see if it resolves your issues, too. Would you mind test it on your device? 🙇

**Reply by lodembeep** [reply]
@taki-on would it be possible to join the testflight ? (I have the same issue IOS 16.1.1 Iphone 8 plus)

**Reply by taki-on** [reply]
@lodembeep Yes :) would you mind sharing your email, with which I can invite you?

**Reply by lodembeep** [reply]
Already found the Testflight invite link, will let you know if the update worked

**Reply by lodembeep** [reply]
1.151.0 seems to have fixed the crash issue for me

**Reply by taki-on** [reply]
Thanks @lodembeep !!!!

**Reply by mallowigi** [reply]
Not me though

**Reply by taki-on** [reply]
We better understood the root cause of the crashes on iOS 16, and we pushed one more beta build with a better fix. Would you mind trying out the latest beta, 1.152.0 (124560519) @lodembeep and @mallowigi ? 🙇

**Reply by lodembeep** [reply]
> We better understood the root cause of the crashes on iOS 16, and we pushed one more beta build with a better fix. Would you mind trying out the latest beta, 1.152.0 (124560519) @lodembeep and @mallowigi ? 🙇 

Updated, working for me, no crash 👍

**Reply by taki-on** [reply]
Thank you a lot @lodembeep ! 👍 👍 👍

**Reply by mallowigi** [reply]
Fixed on my end as well. Thank goodness for test flight

**Comment by justtal** [comment]
Same here. App crashes. Tried to reinstall and still crashing. iOS 16.1

**Comment by maestrokfir** [comment]
Well? Can we get a fix for this? Some people use the mobile app as the authenticator... not good.

**Comment by travisbaars** [comment]
I am also having this problem on iOS 16.3.1. Just updated the app to version 1.150.0 and it just crashes instantly. Restarted device and deleted/reinstalled app with no luck.

**Comment by wandeep** [comment]
> ### Select Topic Area
> Bug
> 
> ### Body
> Since today, I can no longer open the application (ios 16.2). I tried deleting and reinstalling the app but to no success.
> 
> This is very problematic as GitHub is part of my workflow. Fortunately I can navigate GitHub.com but it's still a major annoyance.

Also experiencing this issue - running on iOS 16.2

**Comment by GuiFlam** [comment]
Same here! iOS 16.2

**Comment by dhrn** [comment]
Same here!. Tried re-install but no Luck, it happening for last 18 hours. iOS 16.3

**Comment by taki-on** [comment]
We can reproduce the crash. We are actively working on a fix and we will keep you updated. Thank you for bearing with us!

**Comment by taki-on** [comment]
We got a fix and verified it with the latest beta build. The fix will go out with the next release, v1.151.0, which will be available soon after an Apple App Store review. Sorry for the inconvenience and thank you for bearing with us!

**Comment by maestrokfir** [comment]
@taki-on I'm really surprised that a software company with a billion in annual revenue can release an update on the Apple store that causes an immediate crash (open app launch) for a significant portion of their users.

This is something that 3-man startup might do. Aren't there staging/testing processes in place that prevent this sort of thing? Isn't there a team in your billion dollar company that worries about nothing except making sure new builds don't crash upon launch?

Not to sound snarky, but this sort of thing should be easily avoidable for such an organization.

**Comment by ettaboyle** [comment, marked as answer]
Update from GitHub: GitHub iOS 1.151.0 (with a fix for the crashes on iOS 16) is now live in the Apple App Store.

**Comment by y57** [comment]
nice

**Comment by Endermanbugzjfc** [comment]
nice

**Comment by Elue-dev** [comment]
This is really embarrassing at this point, i have same issue right now after updating to the latest version, are you guys saying we should never just update our app?

you guys need to do better.

Github version: 1.227.0
iOS device: iPhone 13 pro, iOS 26

**Reply by Elue-dev** [reply]
why is there no response on this from the github team!????

**Reply by nguyentandat1302** [reply]
great! please make my answer the solution to this post. I'm looking for badge
